src/sentry/issues/status_change_consumer.py

```diff
@@ -145,16 +145,14 @@ def bulk_get_groups_from_fingerprints(
         (project_id, fingerprint[0]) for project_id, fingerprint in project_fingerprint_pairs
     }
     for project_id, fingerprint in fingerprints_set - found_fingerprints:
-        try:
-            raise Exception("grouphash.not_found")
-        except Exception:
-            logger.exception(
-                "grouphash.not_found",
-                extra={
-                    "project_id": project_id,
-                    "fingerprint": fingerprint,
-                },
-            )
+        metrics.incr(
+            "occurrence_ingest.grouphash.not_found",
+            sample_rate=1.0,
+            tags={
+                "project_id": project_id,
+                "fingerprint": fingerprint,
+            },
```

Comment:
This will have too high cardinality, as a rule of thumb if there's more than 100 or so unique values it shouldn't be a tag in DataDog.

Project id might be OK-ish, but fingerprint for sure isn't.

Reply:
see: https://www.notion.so/sentry/Cardinality-made-easy-aba4f9cd0f5e4f3488c6d9413bc7239d